src/components/ChatSearch/_chat-search.scss

```
@@ -0,0 +1,32 @@
+.chat-search {
+  position: relative;
+}
+
+.chat-search__input {
+  width: 100%;
+  padding: 14px 16px 14px 48px;
+  border: none;
+  border-radius: 16px;
+  outline: 2px solid transparent;
+  font-size: 18px;
+  transition: outline-color var(--transtition-duration);
+
+  &::placeholder {
+    color: var(--placeholder-color);
+  }
+
+  &:hover {
+    outline-color: var(--border-hover-color);
+  }
+
+  &:focus {
+    outline-color: var(--border-focus-color);
+  }
+}
+
+.chat-search__icon {
+  position: absolute;
+  left: 16px;
+  top: 50%;
+  transform: translateY(-50%);
+}
```

Рекомендую обратить внимание на  удобства scss, которые прекрасно сочетаются с БЭМ именованием классов и упрощают написание и чтение css
```suggestion
.chat-search {
  position: relative;

  &__input {
    width: 100%;
    padding: 14px 16px 14px 48px;
    border: none;
    border-radius: 16px;
    outline: 2px solid transparent;
    font-size: 18px;
    transition: outline-color var(--transtition-duration);

    &::placeholder {
      color: var(--placeholder-color);
    }

    &:hover {
      outline-color: var(--border-hover-color);
    }

    &:focus {
      outline-color: var(--border-focus-color);
    }
  }

  &__icon {
    position: absolute;
    left: 16px;
    top: 50%;
    transform: translateY(-50%);
  }
}
```

---

Здесь я осознанно не применяю такой подход. Не люблю эти вложенности через амперсанд, так как потом трудно искать нужный селектор. Использую только для псевдоэлементов(-классов) или модификаторов.